docs/plantuml/shadow_design_typicaloperation.pu

```diff
@@ -3,21 +3,18 @@ skinparam classFontSize 8
 skinparam classFontName Helvetica
 autonumber
 
-box "JSON Library" #LightSalmon
-    participant "JSON" as json
-end box
-
-box "Shadow Library" #LightBlue
-    participant "Shadow" as shadow
-end box
-
 box "Application" #LightGreen
     participant "Application" as application
     participant "callback" as callback
```

Comment:
Should we capitalize Callback too since all the other life lines are?

Reply:
correct! will modify it.